src/components/Canvas/index.ts

```diff
@@ -1404,6 +1404,11 @@ export default class Canvas extends EventDispatcher {
     const mouseCartCoord = this.getMouseCartCoord(evt);
     const pixelIndex = this.getPixelIndexFromMouseCartCoord(mouseCartCoord);
     if (pixelIndex) {
```

Comment:
I think we do not necessarily need to show cursor style only when the cursor is in the pixel canvas grid. I think we can simply show cursor when the user's mouse enters the canvas html element!